packages/vm/test/api/buildBlock.spec.ts

```diff
@@ -157,7 +158,7 @@ describe('BlockBuilder', () => {
         epoch: 30000,
       },
     }
-    const defaultChainData = {
+    const defaultChainData: Omit<GethGenesis, 'alloc'> = {
```

Comment:
Could we also default the `alloc: {}` here instead of the `Omit`?

Reply:
Sure that's a nice workaround & simpler, will adjust